Documents the Kusama Time Warp (#824)
w3f/polkadot-wiki: docs/kusama-timeline.md

took place at 22:45 CET.

> Currently, Kusama is a healthy Proof-of-Stake network with over 150 validators and over a million

Reviewer: ```suggestion
Currently, Kusama is a healthy Proof-of-Stake network with over 500 validators and over two million
```

available. Notably, the Sudo key still existed and was used to initiate further upgrades. Balance
transfers were still disabled for a short while.

Kusama now has its full functionality enabled.

## Kusama's First Adventure

[Source](https://polkadot.network/kusamas-first-adventure/)

> On 4th January 2020, the Polkadot mainnet runtime, which was still at that time not live, was
> uploaded to the Kusama chain during a runtime upgrade. The mishap was due to a recent split apart of
> the Kusama logic from the Polkadot logic that was not properly named. This led to a halt of block
> production on the Kusama chain and bricked the chain entirely.

Reviewer: ```suggestion
On 4th January 2020, the Polkadot mainnet runtime, which at that time still wasn't live, was
uploaded to the Kusama chain during a runtime upgrade. The mishap was due to a recent split of
the Kusama logic from the Polkadot logic and that runtime was not properly named. This led to a 
halt of block production on the Kusama chain and bricked the chain entirely.
```

The solution to the issue involved a rollback of the chain history before the problematic runtime
upgrade took place. However, due to intricacies of the block production mechanism, it was also
necessary to encapsulate the validators of the chain into a time bubble to trick them into believing
that they were producing blocks in the past. Furthermore, in order for the chain to catch up to the

> present moment it was necessary to make time exist in the bubble at a speed of six times greater

Reviewer: ```suggestion
present moment it was necessary to make time flow in the bubble at a speed of six times greater
```
